### ISSUE TITLE
Hardcoded webdriver-manager version

### ISSUE THREAD

**Issue by SmithPeder** [webkom/vote#492]
## Hardcoded chrome version
<hr>

### TL;DR
As per https://github.com/webkom/vote/pull/491 we need to update the hardcoded value listed in the `package.json` to match the chrome version

### Issue

When we run protractor we manually list the chrome-version in the package.json
https://github.com/webkom/vote/blob/83302e6ed02309a0124cdd43dc8e1658ab2688c3/package.json#L19

When this runs through our drone pipeline it runs on the circleci/node:16-browsers image. We don't know when circle updates their chrome version so the pipeline can suddenly fail. 
https://github.com/webkom/vote/blob/83302e6ed02309a0124cdd43dc8e1658ab2688c3/.drone.yml#L66

One can see what version of google chrome the image with
```sh
$ docker run -it --entrypoint "google-chrome" circleci/node:16-browsers --version
Google Chrome 96.0.4664.110
```

This is sub-optimal and can be annoying. Should find a better way to run this.
